postproengine/reynoldsstress.py

```diff
@@ -0,0 +1,133 @@
+# Get the location where this script is being run
+import sys, os
+scriptpath = os.path.dirname(os.path.realpath(__file__))
+basepath   = os.path.dirname(scriptpath)
+# Add any possible locations of amr-wind-frontend here
+amrwindfedirs = ['../',
+                 basepath]
+for x in amrwindfedirs: sys.path.insert(1, x)
+
+from postproengine import registerplugin, mergedicts, registeraction, contourplottemplate
+import postproamrwindsample_xarray as ppsamplexr
+import postproamrwindsample as ppsample
+import numpy as np
+import pickle
+import matplotlib.pyplot as plt
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+
+"""
+Plugin for creating Reynolds stress averages on sample planes
+
+See README.md for details on the structure of classes here
+"""
+
+@registerplugin
+class postpro_averageplanes():
```

Comment:
I'm confused, are you added this as an action to the existing averaged planes executor, or is this a separate executor? Does it make sense as an action to the average or instantaneous executor?